src/main/java/uk/gov/hmcts/ccd/domain/service/createevent/ClassifiedCreateEventOperation.java

```diff
@@ -26,8 +26,7 @@ public ClassifiedCreateEventOperation(@Qualifier("default") CreateEventOperation
     @Override
     public CaseDetails createCaseEvent(String caseReference,
                                        CaseDataContent content) {
-        final CaseDetails caseDetails = createEventOperation.createCaseEvent(caseReference,
-                                                                           content);
```

Comment:
Could you please take developed  version  and exclude this file from the pr? unless checkstyle crash :)

Reply:
reverted to develop version